daemon/audit.c

```diff
@@ -457,27 +472,14 @@ audit_send_next_stored(const container_t *c)
 			return -1;
 		}
 
-		container_audit_set_last_ack(c, NULL);
+		container_audit_set_last_ack(c, "");
```

Comment:
c_audit_set_last_ack frees the last ack string,
thus it should be heap allocated